bin/daenerys

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env php
+<?php
+function includeIfExists($file) 
+{
+    if (file_exists($file)) {
+        return include $file;
+    }
+}
+
+// Dance to find the autoloader.
+// TOOD: change this to open up the Composer config and use $c['config']['vendor-dir'] instead of "vendor"
+includeIfExists(getcwd() . '/vendor/autoload.php') ||
+includeIfExists(__DIR__ . '/../vendor/autoload.php') ||
+includeIfExists(__DIR__ . '/../autoload.php');
```

Comment:
The most common use case is inside a crate, w/ the core installed as a dependency off packagist. In that case, you'd be running daenerys from `crate/`, and then `getcwd() . '/vendor/autoload.php'` is the same as `__DIR__ . '/../vendor/autoload.php'`.

But, if you installed the core off a local copy, then those directories are different, and the current working directory is the one you want, not the vendor inside of your `/path/to/core/`.